clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/description/UpsertAmazonLoadBalancerClassicDescription.java

```diff
@@ -84,7 +86,7 @@ public void setCrossZoneBalancing(Boolean crossZoneBalancing) {
     this.crossZoneBalancing = crossZoneBalancing;
   }
 
-  private Boolean crossZoneBalancing = Boolean.TRUE;
+  private Boolean crossZoneBalancing;
```

Comment:
the default value for this is now in `aws.deployDefaults.loadBalancing` (and defaulted to true if unconfigured)